tg/controllers.py

```diff
@@ -34,6 +34,8 @@
 # cause blan areas on the msg display screen
 MSGS_LEFT_SCROLL_THRESHOLD = 2
 
+key_bind_handler = Callable[[Any], Any]
```

Comment:
bad type name, should be `key_bind_handler_type`

Reply:
👌